Changes

```diff
@@ -37,14 +37,17 @@ Working version
   (Stephen Dolan, Xavier Leroy and David Allsopp,
    review by Xavier Leroy and Gabriel Scherer)
 
-* #8711: The major GC hooks are no longer allowed to interact with the
+- #8711: The major GC hooks are no longer allowed to interact with the
```

Comment:
Is this meant to be here?

Reply:
Yes. Makes the Changes file consistent. Other items use `-` while this one uses `*`. Happy to remove it.

Reply:
The asterisk denotes the fact it's a breaking change (see [the top of the file](https://github.com/ocaml/ocaml/blob/trunk/Changes#L4))

Reply:
Didn't know that. Reverted this change now.